tests/unit/materializers/test_default_materializer.py

```diff
@@ -0,0 +1,28 @@
+#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+
+from tests.unit.test_general import _test_materializer
+
+
+class Unmaterializable:
+    """A class that has no materializer."""
+
+    cat = "aria"
+
+
+def test_default_materializer(clean_client):
```

Comment:
If you also get around to adding in the validation for mismatches of Python versions, would be nice to have a test here that confirms that's also happening.